pkg/acsengine/types.go

```diff
@@ -101,3 +103,86 @@ type KeyVaultRef struct {
 }
 
 type paramsMap map[string]interface{}
+
+// CreateMockContainerService returns a mock container service for testing purposes
+func CreateMockContainerService(containerServiceName, orchestratorVersion string, masterCount, agentCount int, certs bool) *api.ContainerService {
```

Comment:
Let's move this func to another file, perhaps `mocks.go`?

Reply:
done